pkg/scheduler/cache/cache.go

```diff
@@ -1,7 +1,9 @@
 package cache
 
 import (
-	"sync"
+	lister "github.com/karmada-io/karmada/pkg/generated/listers/cluster/v1alpha1"
```

Comment:
nit: move down and group with karmada pkg

Reply:
Done